eng/scripts/typespec/Generate-Code.ps1

```diff
@@ -52,9 +52,8 @@ Invoke-LoggedCommand "dotnet build $packageRoot/generator/TestProjects/Local/Bas
 Write-Host "Generating MgmtTypeSpec" -ForegroundColor Cyan
 Invoke-LoggedCommand (Get-TspCommand "$mgmtTypespecTestProject/main.tsp" $mgmtTypespecTestProject)
 
-# temporarily disable building MgmtTypeSpec because now the generated code of this project cannot build
-# Write-Host "Building BasicTypeSpec" -ForegroundColor Cyan
-# Invoke-LoggedCommand "dotnet build $packageRoot/generator/TestProjects/Local/Mgmt-TypeSpec/src/MgmtTypeSpec.csproj"
+Write-Host "Building BasicTypeSpec" -ForegroundColor Cyan
```

Comment:
Why do we have a Generate-Code.ps1 here?  Shouldn't we be using /eng/packages/http-client-csharp/eng/scripts/Generate.ps1 which follows the pattern of ms/tsp as well?

Reply:
`/eng/packages/*/eng/scripts` was moved to `/eng/scripts/emitter` in https://github.com/Azure/azure-sdk-for-net/pull/46511 by @hallipr.
Since we have more logic in /eng/packages/http-client-csharp/eng/scripts/Generate.ps1 now, I will add Mgmt-TypeSpec generation to it in this PR.
Then, we can sync with @hallipr if it should be moved to `/eng/scripts/emitter`

Reply:
As offline discussed, will deprecate `eng/scripts/tyepspec/Genrate-Code.ps1` in a later PR.